[#736] Auto Accepted Community Feature (connects #736)

nirojdyola opened the pull request.

nirojdyola committed:
[#736] Auto Accepted Community Feature

dogi reviewed (changes requested):
@nirojdyola if I understand you are autoaccepting every community every time ...

maybe I am wrong ;)
the issue addresses something different

like we add something to configuration doc so that this autoaccepting will happen

nirojdyola committed:
[#736] auto accepted Community

nirojdyola committed:
[#736] deleted unwanted code

nirojdyola commented:
While deleting a document of  one environment from another environmnet,  I got the problem CORS. Then, I changed the CORS config in the couch db configurations :
In the cors section I have added 
headers : accept, authorization, content-type, origin, referer, cache-control, x-requested-with
methods: GET, PUT, POST, HEAD, DELETE
origins	: *
I have changed in the environment http://ht.np.ole.org:5982 .

nirojdyola commented:
![autoaccept](https://cloud.githubusercontent.com/assets/7495421/26298081/8a942af8-3ef5-11e7-98b0-c0dde130caed.png)

Rupesh87 committed:
[#736] checkbox and accept label at same position

nirojdyola committed:
[#736] login fixed after autoaccept

nirojdyola committed:
[#736] removed white space in Home Urdu and Home.css

nirojdyola committed:
[#736] removed community models included in index.html
